<launch>

  <!-- Definir los parámetros predeterminados -->
  <arg name="protocol_version" default="2.0" />
  <arg name="baudrate" default="57600" />
  <arg name="port_name_0" default="/dev/ttyUSB0" />
  <arg name="dmxl_id_0" default="" />
  <arg name="port_name_1" default="/dev/ttyUSB1" />
  <arg name="dmxl_id_1" default="" />
  <!--<arg name="port_name_2" default="/dev/ttyUSB2" />
  //<arg name="dmxl_id_2" default="" />-->

  <group ns="$(arg namespace)">
    <!-- Nodo para ejecutar testPositionControl desde dynamixel_ros_library -->
    <node  
      pkg="wrist_control"
      type="wrist_orientation"
      name="wrist_orientation_node"
      output="screen">

      <!-- Pasar los parámetros como argumentos -->
      <param name="PROTOCOL_VERSION" value="$(arg protocol_version)" />
      <param name="BAUDRATE" value="$(arg baudrate)" />
      <param name="PORT_NAME_0" value="$(arg port_name_0)" />
      <param name="DMXL_ID_0" value="$(arg dmxl_id_0)" />
      <param name="PORT_NAME_1" value="$(arg port_name_1)" />
      <param name="DMXL_ID_1" value="$(arg dmxl_id_1)" />
      <!--<param name="PORT_NAME_2" value="$(arg port_name_2)" />
      <param name="DMXL_ID_2" value="$(arg dmxl_id_2)" />-->

    </node>
  </group>
</launch>